Missionframework/modules/11_permission/fnc/fn_permission_addPermissionHandler.sqf

```diff
@@ -0,0 +1,62 @@
+/*
+    KPLIB_fnc_permission_addPermissionHandler
+
+    File: fn_permission_addPermission.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-12-09
+    Last Update: 2018-12-27
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Adds a new permission to the permission system.
+
+    Parameter(s):
+        _permission     - Permission name                               [STRING, defaults to ""]
+        _condition      - Code which is executed on permission check    [CODE, defaults to {false}]
+        _string         - Permission stringtable path                   [STRING, defaults to ""]
+        _default        - Default permission                            [BOOL, defaults to false]
+        _group          - Permission group name                         [STRING, defaults to "Misc"]
+        _groupString    - Permission group stringtable path             [STRING, defaults to "STR_KPLIB_PERMISSION_GROUPMISC"]
+        _vehCheck       - Vehicle clasnames for the check               [ARRAY, defaults to []]
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
```

Comment:
This function is also intended to just run on server?
Suggest to also add a failsave with `if (!isServer) exitWith {}`.
Just for the case that someone adds a module and tries to add a permission in a "non-server scope".